samples/dymaptic.GeoBlazor.Core.Sample.Shared/Pages/LayerLists.razor

```diff
@@ -55,10 +60,7 @@
     private Task<ListItem> OnListItemCreated(ListItem item)
     {
         item.Visible = true;
-        if (item.Title?.StartsWith("Transit") ?? false)
-        {
-            item.Title = $"Mass {item.Title}";
-        }
+
```

Comment:
I don't think we wanted to delete this `if` block, it was demonstrating the purpose of overriding `OnListItemCreated`.

Reply:
I added this back in, going to look for a similar example for the basemap layerlist